glfs.c

```diff
@@ -448,29 +449,29 @@ static int tcmu_glfs_open(struct tcmu_device *dev)
 		goto fail;
 	}
 
-	gfsp->fs = tcmu_create_glfs_object(config, &gfsp->hosts);
+	gfsp->fs = tcmu_create_glfs_object(dev, config, &gfsp->hosts);
 	if (!gfsp->fs) {
-		tcmu_err("tcmu_create_glfs_object failed\n");
+		tcmu_dev_err(dev, "tcmu_create_glfs_object failed\n");
 		goto fail;
 	}
 
 	gfsp->gfd = glfs_open(gfsp->fs, gfsp->hosts->path, ALLOWED_BSOFLAGS);
 	if (!gfsp->gfd) {
-		tcmu_err("glfs_open failed: %m\n");
+		tcmu_dev_err(dev, "glfs_open failed: %m\n");
 		goto unref;
 	}
 
 	ret = glfs_lstat(gfsp->fs, gfsp->hosts->path, &st);
 	if (ret) {
-		tcmu_err("glfs_lstat failed: %m\n");
+		tcmu_dev_err(dev, "glfs_lstat failed: %m\n");
 		goto unref;
 	}
 
 	if (st.st_size != tcmu_get_device_size(dev)) {
-		tcmu_err("device size and backing size disagree: "
-		       "device %lld backing %lld\n",
-		       tcmu_get_device_size(dev),
-		       (long long) st.st_size);
+		tcmu_dev_err(dev, "device size and backing size disagree: "
+		                  "device %lld backing %lld\n",
+		                  tcmu_get_device_size(dev),
+		                  (long long) st.st_size);
```

Comment:
Waiting for mike ACK on below, which is important for this patch.
Once mike got convinced with my comment below, will change this for you :-)